interpreter/terminal_interface/magic_commands.py

```diff
@@ -38,8 +40,9 @@ def handle_help(self, arguments):
       "%debug [true/false]": "Toggle debug mode. Without arguments or with 'true', it enters debug mode. With 'false', it exits debug mode.",
       "%reset": "Resets the current session.",
       "%undo": "Remove previous messages and its response from the message history.",
-      "%save_message [path]": "Saves messages to a specified JSON path. If no path is provided, it defaults to 'messages.json'.",
-      "%load_message [path]": "Loads messages from a specified JSON path. If no path is provided, it defaults to 'messages.json'.",
+      "%save_message [path]": "Saves messages to a specified JSON path. If no path is provided, it defaults to '[date]_messages_[counter].json'.",
+      "%load_message [path]": "Loads messages from a specified JSON path. If no path is provided, it loads the latest default message '[date]_messages_[counter].json'.",
```

Comment:
This is a minor rendering thing, but I think the underscores (`_`) here need to be escaped with `\_` or they try to render as markdown and end up italicized.

### Before escaping underscores in filenames

![Screen Shot 2023-10-08 at 11 55 58 AM](https://github.com/KillianLucas/open-interpreter/assets/1667415/5594555b-7480-4808-bee1-01bc45531efe)

### With filename underscores escaped in `%load_message` help text

![Screen Shot 2023-10-08 at 11 57 15 AM](https://github.com/KillianLucas/open-interpreter/assets/1667415/c80ebca3-5bb0-406d-a144-62a6a68fb0d1)

Reply:
Good spot - will update!